robotiq_s_model_articulated_gazebo/package.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0"?>
+<package>
+  <name>robotiq_s_model_articulated_gazebo</name>
+  <version>1.0.0</version>
+  <description>Launch files for spawning the robotiq s-model articulated gripper</description>
+
+  <maintainer email="dash@clearpathrobotics.com">Devon Ash</maintainer>
+  <author email="dash@clearpathrobotics.com">Devon Ash</author>
+
+  <license>BSD</license>
```

Comment:
That's right, I'll have to go and add the Apache license to the RobotiqHandPlugin.cpp and mention any modifications.